pkg/driver/bigquery/test_cred.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "test-project",
+  "private_key_id": "someprivatekey",
+  "private_key": "\\n-----END PRIVATE KEY-----\n",
```

Comment:
Should that first `\\n` be `\n`?

Reply:
now that we are don't have a `bigquery_client.go` we don't really need this and much more simplified. will remove this.